### ISSUE TITLE
Channel not configured - Thing offline

### ISSUE THREAD

**Issue by htreu** [eclipse/smarthome#5898]
In my setup I have configured a KNX actor with 4 switch channels using the Thing DSL like this:

```
Thing device siemens_4_switch_1_1_32 {
    Type switch : kanalA "Licht_Bad_DG" [ ga="<4/1/1+14/1/1" ]
    Type switch : kanalB
    Type switch : kanalC "Router_DG"    [ ga="<4/3/1+14/3/1" ]
    Type switch : kanalD
}
```

The idea is to model all devices completely regardless if channels are in use or not. With the current behaviour in ESH this leads to the thing being in the status `OFFLINE - HANDLER_CONFIGURATION_PENDING`, since the required channel configuration `ga` for channels `kanalB` & `kanalD` are not fulfilled.

I'm wondering wether this is intended or if it is sufficient to log a warning (or give more details to the user in PaperUI) and proceed with the handler initialization.

**Comment by sjka** [comment]
> I'm wondering wether this is intended

Yes, that actually was precisely the idea of #5175.

> or if it is sufficient to log a warning

That would also be a possibility, of course.

My intention was to at least not let the binding fail silently or load off the checks and according logging to _all_ bindings. That would actually be the case again if this check gets degraded to logging a warning only and still proceeding with initializing the handler again: binding implementations suddenly have to cope again with missing configuration parameters which they actually declared to be mandatory. If they could rely on them being present, the code looks much cleaner IMHO.

**Comment by htreu** [comment]
> maybe it would also be worth enhancing it by providing some clue what configuration parameters are missing as an alternative?

Absolutely! When I understand the code in `ThingManager` correct there is actually room for enhancement. A message in PaperUI like `Channel "kanalB" is missing required parameter "ga"` would be nice.
